### PR TITLE
Add u1 type to ngraph python api

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
 - Add missing u1 type to ngraph python api
 
### Tickets:
 - 49517

### CODE REVIEW

File: ngraph/python/tests/test_ngraph/test_create_op.py [hunk 1]
```diff
@@ -49,8 +47,8 @@ def test_binary_convolution(dtype):
     input1_shape = [1, 1, 3, 3]
     expected_shape = [1, 1, 7, 7]
 
-    parameter_input0 = ng.parameter(input0_shape, name="Input0", dtype=dtype)
-    parameter_input1 = ng.parameter(input1_shape, name="Input1", dtype=dtype)
+    parameter_input0 = ng.parameter(input0_shape, name="Input0", dtype=np.float32)
+    parameter_input1 = ng.parameter(input1_shape, name="Input1", dtype=ng.utils.types.u1)
```
Comment (jiwaszki) [inline ngraph/python/tests/test_ngraph/test_create_op.py:51]: @postrational what do you think about this name?

**Review by jdanieck — APPROVED** [review]

File: ngraph/python/src/ngraph/utils/types.py [hunk 2]
```diff
@@ -33,6 +33,9 @@
 ScalarData = Union[int, float]
 NodeInput = Union[Node, NumericData]
 
+# Custom dtype u1 to distinguish uint8 and u1 data
+u1 = np.dtype([("data", np.uint8, 1)])
```
Comment (postrational) [inline ngraph/python/src/ngraph/utils/types.py:37]:
```suggestion
u1 = np.dtype((np.uint8, (1,)))
```

**Review by tsocha — APPROVED** [review]

File: ngraph/python/src/ngraph/utils/types.py (hunk 2 above)
Comment (mitruska) [reply ngraph/python/src/ngraph/utils/types.py]: Done

File: ngraph/python/tests/test_ngraph/test_create_op.py (hunk 1 above)
Comment (mitruska) [reply ngraph/python/tests/test_ngraph/test_create_op.py]: I've added import to ngraph module `__init__` file so now it's avaible as `ng.packed_u1`.

**Review by jiwaszki — APPROVED** [review]

**Comment by jiwaszki** [comment]
Closing. Creating more generic packed types conversion/casting in other PR.

**Pull request closed without merging**